mesonbuild/compilers/compilers.py

```diff
@@ -815,7 +816,7 @@ def compile(self, code: 'mesonlib.FileOrString',
             if extra_args:
                 commands += extra_args
             # Generate full command-line with the exelist
-            command_list = self.get_exelist() + commands.to_native()
+            command_list = self.get_exelist(ccache=not no_ccache) + commands.to_native()
```

Comment:
I was pondering whether, below, to drop CCACHE_DISABLE=1.

Then I realized we should definitely still set this, because people could do `CC=/usr/lib/ccache/bin/cc` and we might as well suppress it with an environment variable if we can, and the tool respects it.

Reply:
yeah, doesn't hurt to keep it.